python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -1354,9 +1363,18 @@ def get() -> "OpExecutionContext":
         return ctx.get_op_execution_context()
 
 
+def _copy_docs_from_op_execution_context(obj: T_Annotatable) -> T_Annotatable:
```

Comment:
is there any use in this being a more general purpose decorator? If so I could move it to `_annotations.py`

Reply:
I don't think so-- and strictly speaking it isn't an annotation method (at least as I intended the term). I think to simplify I would remove `_get_annotation_target` and `isfunction` here, pretty sure you don't need them-- though I would check the docs rendering or maybe just add a line to your test that ensures docstrings match.

Reply:
ok good to know. went ahead and added the test!